src/api/stats.ts

```diff
@@ -52,22 +53,22 @@ export interface DefaultStats {
 }
 
 const BASE_QUERY = `
-            catalog_name,
-            grain,
-            ts
+    catalog_name,
+    grain,
+    ts
 `;
 
-const DEFAULT_QUERY = `    
-            ${BASE_QUERY},
-            bytes_written_by_me,
-            docs_written_by_me,
-            bytes_read_by_me,
-            docs_read_by_me,
-            bytes_written_to_me,
-            docs_written_to_me,
-            bytes_read_from_me,
-            docs_read_from_me
-        `;
+const DEFAULT_COLS = [
+    'bytes_written_by_me',
+    'docs_written_by_me',
+    'bytes_read_by_me',
+    'docs_read_by_me',
+    'bytes_written_to_me',
+    'docs_written_to_me',
+    'bytes_read_from_me',
+    'docs_read_from_me',
+];
```

Comment:
Thought I might need to do some query "magic" but didn't. Still leaving this as this pattern seems the most flexible.